src/languages/haxe.js

```diff
@@ -10,14 +10,17 @@ export default function(hljs) {
   const IDENT_RE = '[a-zA-Z_$][a-zA-Z0-9_$]*';
   const IDENT_FUNC_RETURN_TYPE_RE = '([*]|[a-zA-Z_$][a-zA-Z0-9_$]*)';
 
+  // C_NUMBER_RE with underscores and literal suffixes
+  const HAXE_NUMBER_RE = '(-?)(\\b0[xX][a-fA-F0-9_]+|(\\b\\d+(\\.[\\d_]*)?|\\.[\\d_]+)(([eE][-+]?\\d+)|i32|u32|i64|f64)?)'
```

Comment:
Please write full regex as a regex literal, not a string.

Reply:
👍